### ISSUE TITLE
More options in dialogs

### ISSUE THREAD

**Issue by genitr** [AscensionGameDev/Intersect-Engine#587]
**Is your feature request related to a problem? Please describe.**
At the moment, the editor supports only 4 options for choosing in dialogs.  In my opinion, this is pretty small.
**Describe the solution you'd like**
I propose to expand this function to 8 options, while keeping the individual settings as it was done in the case of one option, with two options, etc.

**Additional context**
Why do I think you need at least 8 options in dialogs?  For example, 6 options are enough for dialogues with NPCs.  Templates with 7 and 8 options can be used to create a simple menu that is not in the editor, but may be needed in the game. For example, the menu "Guilds", "Select a location to move", "Book of Monsters", etc.

**Comment by Cheshire92** [comment]
Might be a little more involved than you'd think.
Personally, if this were to happen I'd prefer it to not be capped at all and just allow you to keep adding options as required. 

Then still have the layout options as they are now with an additional one that has an ever increasing list of options with a scrollbar.

**Comment by genitr** [comment]
> Might be a little more involved than you'd think.
> Personally, if this were to happen I'd prefer it to not be capped at all and just allow you to keep adding options as required.
> 
> Then still have the layout options as they are now with an additional one that has an ever increasing list of options with a scrollbar.

In principle, this is also a good option, perhaps even optimal (for the user).  However, this option undoubtedly looks more complicated in terms of adding to the engine.

**Comment by Cheshire92** [comment]
Yes, however long term it would likely be the better option rather than getting another issue in a year's time from someone needing 9 options.

**Comment by genitr** [comment]
> Yes, however long term it would likely be the better option rather than getting another issue in a year's time from someone needing 9 options.

I agree with you.  Then, perhaps the best solution would be to add the appropriate attribute to the configuration file, thereby allowing the user to determine how many options are needed for his game.  In the editor, as you said, you can use the scroll bar for a lot of options.
